docs/architecture/stack/evm-state-manager/index.mdx

```diff
@@ -1,31 +1,251 @@
 ---
-title: EVM State Manager
+title: EVM state manager
+description: How state management works on Linea
 sidebar_position: 4
 image: /img/socialCards/evm-state-manager.jpg
 ---
 
-## EVM State Manager
+The state manager is the part of the execution client responsible for updating the state of the 
+network globally, and the state of every account individually. The state manager also audits the 
+"read" access made in the EVM.
 
-### What is it?
+:::info
 
-The part of the execution client responsible for updating the state of the network globally, and the state of every account individually.
+"State", in this context, refers to the data stored on the blockchain at any given point in time. To
+update state is to update the record of the contents of every account — if the contents have 
+changed. 
 
-### What does it do?
+:::
 
-Receives blocks that have been executed by the sequencer, and uses the trace data from their execution to update the state of the network. It then passes this updated network state information to the prover, for subsequent submission to Ethereum
+Its main task is to receive blocks that have been executed by the sequencer and use the trace data 
+from their execution to update the state of the network. The state manager uses a variation of the 
+Merkle tree called a sparse Merkle tree (SMT) to track, manage, and update storage slots representing 
+accounts more efficiently. 
 
-### How does it do it?
+It then passes this updated network state information to the [prover](/architecture/stack/trace-expansion-proving/proving) 
+in the form of Merkle proofs for submission to Ethereum Mainnet (L1). 
 
-Through the Promethean power of cryptography and the careful stewardship of Merkle trees 😝
+Below, we'll explain the element of Linea's state management in greater detail.
 
-The state of every account in the zkEVM is represented by a hash: a unique, encrypted and brief identifier. Because of the way hashes work, any change in the state of an account will result in a changed, but still unique and encrypted, hash.
+## Merkle trees
 
-The relationship between accounts–which accounts control which ones, for example–is represented by a tree structure. The way a small twig is derived from a larger branch, and that from a trunk: in this way, the entire bifurcating, iterative history of the network state is retained.
+### Overview: how Merkle trees work
```

Comment:
This level 3 heading is probably not required? Content can live under the "Merkle trees" topic.